src/main/resources/assets/forge/lang/en_us.json

```
@@ -178,5 +178,8 @@
   "generic.reachDistance": "Reach Distance",
 
   "fluid.minecraft.milk": "Milk",
-  "fluid.minecraft.flowing_milk": "Milk"
+  "fluid.minecraft.flowing_milk": "Milk",
+
+  "forge.experimentalsettings.tooltip": "This world uses vanilla's Experimental Worldgen Settings, which could stop working at any time.",
```

I'm a bit hesitant calling it "vanilla's Experimental Worldgen Settings", since that implies these experimental settings are caused by vanilla MC rather than any added mods/datapacks. I'd rather this lose the "vanilla's" part, as well as the "worldgen" part to match the vanilla messages:
```suggestion
  "forge.experimentalsettings.tooltip": "This world uses experimental settings, which could stop working at any time.",
```
I believe @Commoble originally suggested this wording in Discord, so I am asking for your opinion on this as well.